mysql-test/suite/galera/r/MW-369.result

```diff
@@ -258,6 +258,7 @@ SET SESSION DEBUG_SYNC = 'now SIGNAL signal.wsrep_apply_cb';
 SET GLOBAL DEBUG_DBUG = "";
 SET DEBUG_SYNC = 'RESET';
 connection node_1;
+call mtr.add_suppression("WSREP: Foreign key check table:.*");
```

Comment:
`.*` at the end of the regular expression is redundant, because there is no implied `$` anchor. Shouldn’t there also be a `search_pattern_in_file.inc` for ensuring that the message is present?